theforeman.org/yaml/jobs/test_develop_pr_core.yaml

```diff
@@ -1,42 +0,0 @@
-# Test Foreman PRs, runs Foreman's own unit tests
-- job:
-    name: test_develop_pr_core
-    project-type: matrix
-    concurrent: true
-    properties:
-      - github_foreman
-      - tfm-pull-request-build-discarder
-    wrappers:
-      - abort_build:
-          timeout_min: 240
-    scm:
-      - foreman_pr
-    triggers:
-      - github_pr:
-          context: 'foreman'
-    axes:
-      - axis:
-          type: user-defined
-          name: ruby
-          values:
-            - 2.7
-      - axis:
-          type: user-defined
-          name: database
-          values:
-            - postgresql-integrations
-            - postgresql
-      - axis:
-          type: label-expression
-          name: slave
-          values:
-            - fast
-    builders:
-      - shell: !include-raw: scripts/test/test_develop.sh
```

Comment:
Note to self: this is still used in `theforeman.org/yaml/builders/test_develop.yaml` so shouldn't be dropped yet.